modules/component/src/push/push.pipe.ts

```diff
@@ -0,0 +1,61 @@
+import { ChangeDetectorRef, NgZone, Pipe, PipeTransform } from '@angular/core';
+import { NextObserver, Observable, PartialObserver, pipe, Subject } from 'rxjs';
+import { distinctUntilChanged, map, tap, withLatestFrom } from 'rxjs/operators';
+import {
+  CdAware,
+  CoalescingConfig as NgRxPushPipeConfig,
+  remainHigherOrder,
+  STATE_DEFAULT,
+} from '../core';
+
+@Pipe({ name: 'ngrxPush', pure: false })
```

Comment:
I may have brought this up before, but I think this should just be `push$`
```suggestion
@Pipe({ name: 'push$', pure: false })
```

Reply:
I think we should stick with "vendor" prefix of "ngrx" to:

- avoid any potential naming collisions
- be clear where it's coming from
- similar to `ng`

Reply:
That's fine with me

Reply:
So I keep it.

Reply:
These are just my two cents, but shouldn't it be `ngRxPush` since pipe names should be lowerCamelCase? https://angular.io/guide/styleguide#pipe-names